maistra/scripts/update-deps.sh

```diff
@@ -100,6 +100,8 @@ function run_bazel() {
 }
 
 function main() {
+  source /opt/rh/gcc-toolset-9/enable
```

Comment:
This will fail due to `set -u`. So, basically, remove `set -u` and move this to the top of the file.